src/main/java/core/mate/academy/service/TruckProducer.java

```diff
@@ -0,0 +1,12 @@
+package core.mate.academy.service;
+
+import core.mate.academy.model.Machine;
+import core.mate.academy.model.Truck;
+import java.util.List;
+
+public class TruckProducer implements MachineProducer<Truck> {
+    @Override
+    public List<Machine> get() {
```

Comment:
Do not use raw types. Your `get()` method signature should return `List<Truck>` instead of `List<Machine>`. This ensures type safety and adheres to the Java generics best practices.